### PR TITLE
SONAR-14212 Remove no more used Sonar-Version HTTP header

### PR DESCRIPTION
**Pull request by philippe-perrin-sonarsource**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+- SONAR-14212 Remove no more used Sonar-Version HTTP header
```
Comment (jeremy-davis-sonarsource) [inline CHANGELOG.md]: Fix the language: `Remove obsolete Sonar-Version HTTP header`

**Review by jeremy-davis-sonarsource — APPROVED** [review]
LGTM

**Comment by jeremy-davis-sonarsource** [comment]
My comment applies to the commit messages as well, both here and in SQ

**Review by wouter-admiraal-sonarsource — COMMENTED** [review]
LGTM NFV

**Review by wouter-admiraal-sonarsource — APPROVED** [review]

**Pull request merged**